src/charts/tooltip/TooltipComponent.js

```diff
@@ -47,10 +70,44 @@ export default class TooltipComponent extends React.Component {
 
         delete configuration.data;
         delete configuration.chart;
+        delete configuration.children;
 
         return configuration;
     }
 
+    _handleMouseMove(dataPoint, topicColorMap, x, y) {
+        // Update Tooltip State
+        this.setState((state) => ({
+            ...state,
+            dataPoint,
+            topicColorMap,
+            x,
+            y,
+        }));
+
+        if (this.props.children.props.customMouseMove) {
```

Comment:
nit: For the three conditionals here could do destructuring so don't need to repeat all the nested-ness:
`let {children: {props: {customMouseMove}}} = this.props;`